frontend/src/routes/Home/Overview/components/SummaryStatusCard.tsx

```diff
@@ -47,7 +47,7 @@ export function SummaryStatusCard(props: { key: string; title: string; data: Dat
               {data?.loading ? (
                 <Skeleton style={{ marginBottom: '10px', height: '30px' }} />
               ) : data.mainSection.link ? (
-                <Link style={{ fontSize: 24 }} to={data.mainSection.link}>
+                <Link style={{ fontSize: 24, marginBottom: '.5rem' }} to={data.mainSection.link}>
```

Comment:
Applies a marginBottom to title link to make space for underscore.

Before:
<img width="697" height="280" alt="Screenshot 2025-12-02 at 6 38 54 PM" src="https://github.com/user-attachments/assets/943a9feb-e2be-4373-9669-2cab226d7a40" />

After:
<img width="624" height="265" alt="Screenshot 2025-12-02 at 6 45 25 PM" src="https://github.com/user-attachments/assets/287ef345-29ac-4be7-9684-47c35a5ffc79" />